### ISSUE TITLE
Type not found : rx.observers.CheckedObserver

### ISSUE THREAD

**Issue by sonygod** [darmie/wasp#4]
hello ,damie, I can't build your wasp  

src/wasp/Global.hx:3: characters 8-36 : Type not found : rx.observers.CheckedObserver

**Comment by darmie** [comment]
Yes this is a known issue in latest version of Haxe. Please try an older version of Haxe and use the my own RxHaxe fork.

**Comment by darmie** [comment]
@sonygod you can try the latest commit. 

[Please note that WASP has an issue with reading accurate unsigned integers in C# and Java targets , especially during conversion to other integer types]

**Comment by sonygod** [comment]
@darmie  thanks, when Will hashlink available?

**Comment by sonygod** [comment]
lix install github:DoclerLabs/hexLog as hexlog

**Comment by sonygod** [comment]
"
/root/haxe/haxe_libraries/rxhaxe/0.0.0/github/1296766cf0ba020c1c4f020f016deda9c5c903c6/rx/Thread.hx:5: characters 20-33 : Warning : This typedef is deprecated in favor of sys.thread.Thread
/root/haxe/haxe_libraries/rxhaxe/0.0.0/github/1296766cf0ba020c1c4f020f016deda9c5c903c6/rx/Mutex.hx:3: characters 19-31 : Warning : This typedef is deprecated in favor of sys.thread.Mutex

**Comment by sonygod** [comment]
after compile then run,got nothing trace , and also copy *.wasm to ./test

on ubuntu 18.04

**Comment by darmie** [comment]
@sonygod I haven't really setup hexlog properly, I am also wondering why it doesn't log any output. 

I can't really say when hashlink will be available, This project depends on integer types that aren't supported by Haxe, I had to simulate some of them in [Numerix](https://github.com/darmie/numerix). Wasp depends on Numerix to handle unsigned integegers of various length.

**Comment by darmie** [comment]
> after compile then run,got nothing trace , and also copy *.wasm to ./test
> 
> on ubuntu 18.04

You wouldn't get a trace, a `globals.wat` file will be generated if you run the test
